docs/usage/cli.md

```diff
@@ -20,6 +20,14 @@ available from the command line.
 $ npm install --global babel-cli
 ```
 
+Out of the box, Babel (since v6) doesn't perform any transformation, it only validates the syntax. You may want to install [plugins](/docs/plugins) or more likely whole [presets](/docs/plugins/#presets) before going further. For the sake of the example, we'll use the [es2015](/docs/plugins/preset-es2015/) preset.
+
+```sh
+$ npm install babel-preset-es2015
```

Comment:
I can add it if you really want, but I don't think it's necessary, for the same reason you install babel-cli globally rather as a dev-dependency (or even gulp-babel). This is for the sake of trying things on the command line.